tests/notifiers/test-class-configuration-notifier.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @package WPSEO\Tests\Notifiers
+ */
+
+/**
+ * Unit Test Class.
+ */
+class WPSEO_Configuration_Notifier_Test extends WPSEO_UnitTestCase {
+
+	/**
+	 * Tests the notify method when the onboarding wizard notice wont shown.
```

Comment:
Seeing as it's a name, `Onboarding Wizard` should be capitalized. Also, `wont shown` should be `won't be shown`.

Please also rename the test.